Model/Runner.php

```diff
@@ -35,24 +35,29 @@ public function __construct(
         ];
     }
 
-    public function run($diff)
+    public function run($line)
     {
-        foreach ($diff as $line) {
-            $pathInfo = $this->lineProcessor->toPathInfo($line);
+        $result = [];
+        $result['type'] = '';
+        $result['items'] = [];
+        $result['path'] = '';
 
-            if (empty($pathInfo)) {
-                continue;
-            }
+        $pathInfo = $this->lineProcessor->toPathInfo($line);
 
-            foreach ($this->checkers as $type => $checker) {
-                $result = $checker->check($pathInfo);
-                if (!empty($result)) {
-                    $this->result[$type][$result['patched']] = $result['customized'];
-                    continue 2;
-                }
+        if (empty($pathInfo)) {
+            return $result;
+        }
+
+        foreach ($this->checkers as $type => $checker) {
+            $checked = $checker->check($pathInfo);
+            if (!empty($checked)) {
+                $result['type'] = $type;
+                $result['items'] = $checked['customized'];
+                $result['path'] = $pathInfo['fullpath'];
+                continue;
```

Comment:
Don't think we need this `continue;`

Reply:
You're right. This should be a `break` now that the runner receives each line one at a time. The idea is that if one the checkers hits we don't need to check anymore.